pdc/apps/common/filters.py

```diff
@@ -50,6 +52,23 @@ def filter(self, qs, value):
         return qs
 
 
+class MultiValueRegexFilter(MultiValueFilter):
```

Comment:
For this task to filter contacts by component, previous implementation is MultiValueFilter. I think maybe it's better to support multi-value as before so not to break some existing script. The MultiValueFilter is using 'in', so I think the relationship is 'or' for multi-value.